### PR TITLE
Added topnav bar to List and Expenses view

### PR DESCRIPTION
**Pull request by UnluckyBird**
Fixes issue #4

### CODE REVIEW

**Review by andreasalstrup — APPROVED** [review]
Not sure about `"react-native-webview": "13.6.2",` to `"13.2.2"` but works and LGTM

File: package.json [hunk 1]
```diff
@@ -30,10 +31,11 @@
     "react-native": "0.72.6",
     "react-native-gesture-handler": "~2.12.0",
     "react-native-get-random-values": "^1.9.0",
+    "react-native-reanimated": "~3.3.0",
     "react-native-safe-area-context": "4.6.3",
     "react-native-screens": "~3.22.0",
     "react-native-web": "~0.19.6",
-    "react-native-webview": "^13.6.2",
+    "react-native-webview": "13.2.2",
```
Comment (mikejensen0) [inline package.json:38]: Why downgrade?

**Review by mikejensen0 — APPROVED** [review]
Generally LGTM, just wondering about the react-native-web downgrade

File: package.json (hunk 1 above)
Comment (UnluckyBird) [reply package.json]: Fixed

**Pull request merged**